docs/gc.md

```diff
@@ -132,15 +132,35 @@ There are a number of advantages to using arraylets.
+**(Linux on x86-64 and AIX platforms only)**
 
-### `optavgpause` policy
+[`-Xgcpolicy:metronome`](xgcpolicy.md#metronome-aix-linux-x86-only) is designed for applications that require a precise upper bound on collection pause times as well as specified application utilization: the proportion of time that the application is permitted to use, with the remainder being devoted to GC. The realtime GC runs in short interruptible bursts to avoid stop-the-world pauses.
 
-[`-Xgcpolicy:optavgpause`](xgcpolicy.md#optavgpause) uses concurrent mark and sweep phases, which means that pause times are reduced when compared to optthruput, but at the expense of some performance throughput.
+The realtime heap is allocated as a contiguous range of RAM partitioned into small regions of equal size, usually about 64 KB. 
 
-### `optthruput` policy
+As with the `balanced` GC policy, arrays are represented as arraylets with a spine pointing to a series of regions containing the array elements (see [Arraylets: dealing with large arrays](#arraylets-dealing-with-large-arrays) for more information). Note that JNI access to array data might involve reconstituting arraylets as contiguous arrays, which can significantly slow down processing.
 
-[`-Xgcpolicy:optthruput`](xgcpolicy.md#optthruput) is optimized for throughput by disabling the concurrent mark phase, which means that applications will stop for long pauses while garbage collection takes place. You might consider using this policy when high application throughput, rather than short garbage collection pauses, is the main performance goal.
+Each region of the heap is either empty or contains only objects of the same size or an arraylet. This organization simplifies new object allocation and consolidation of free heap space, allowing the realtime collector to maintain GC throughput while supporting a consistent service level.
+
+Although high application utilization is desirable for optimal throughput, the GC must be able to keep up with the application's memory  allocation rate. 
+
+A higher utilization typically requires a larger heap because the GC isn't allowed to run as much as a lower utilization would permit. The relationship between utilization and heap size is highly application dependent, and striking an appropriate balance requires iterative experimentation with the application and VM parameters. You might need to adjust heap size or pause time or target utilization to achieve an acceptable runtime configuration.
+
+See [`-Xgcpolicy:metronome`](xgcpolicy.md#metronome-aix-linux-x86-only) for details of the configuration parameters you can set.
+
+### `optavgpause` and `optthruput` policies
+
+Both of these policies use the global collector to manage a flat heap comprised of a single region. The application runs until the heap is exhausted and the global collector runs to mark and sweep the heap. Additionally, if the heap becomes fragmented, the global collector compacts the heap.
```

Comment:
global Mark-and-Sweep collector 

'runs until the heap is exhausted' is only true for throughput. For optavgpause, a cycle is preemptively started (kicked off) so that the cycle finishes before heap is completely exhausted.

Reply:
Updates made.